lib/ansible/galaxy/data/default/collection/galaxy.yml.j2

```diff
@@ -1,65 +1,11 @@
 ### REQUIRED
-
-# this can be a company/brand/organization or product namespace
-# under which all content lives
-namespace: {{ namespace }}
-
-
-# the designation of this specific collection
-name: {{ collection_name }}
-
-
-# semantic versioning compliant version designation
-version: 1.0.0
-
-# the filename for the readme file which can be either markdown (.md)
-readme: README.md
-
-
-# a list of the collection's content authors
-# Ex: 'Full Name <email> (http://site) @nicks:irc/im/site#channel'
-authors:
-- {{ author }} <example@domain.com>
-
-
-### OPTIONAL but strongly advised
-
-# short summary of the collection
-description: {{ description }}
-
-
-# Either a single valid SPDX license identifier or a list of valid SPDX license
-# identifiers, see https://spdx.org/licenses/. Could also set `license_file`
-# instead to point to the file the specifies the license in the collection
-# directory.
-license: {{ license }}
-
-
-# list of keywords you want to associate the collection
-# with for indexing/search systems
-tags: []
-
-
-# A dict of dependencies. A dependency is another collection
-# this collection requires to be installed for it to be usable.
-# The key of the dict is the collection label (namespace.name)
-# and the value is a spec for the semver version required.
-dependencies: {}
-
-
-### URLs
-
-# url of originating SCM repository
-repository: {{ repository_url }}
-
-
-# url to online docs
-documentation: {{ documentation_url }}
-
-
-# homepage of the collection/project
-homepage: {{ homepage_url }}
-
-
-# issue tracker url
-issues: {{ issue_tracker_url }}
\ No newline at end of file
+{% for option in required_config %}
+{{ option.description | comment_ify }}
+{{ {option.key: option.value} | to_yaml }}
+{% endfor %}
+
+### OPTIONAL but strongly adviced
```

Comment:
Typo here (adviced/advised) drew my eye, but actually I think "recommended" may be better.
```suggestion
### OPTIONAL but strongly recommended
```